docs/operate/get-started/other-hardware/create-module/dependencies.md

```diff
@@ -287,6 +291,287 @@ There is not currently an SDK method to directly access configuration attributes
 You must access the API key module environment variables to establish the app client connection.
 {{% /hiddencontent %}}
 
+## Use SDK methods on dependencies
+
+Once you have added a dependency to your module, you can use SDK methods on the resource client.
+For example:
```

Comment:
I think we have to be careful with this because dependencies are at the model level
```suggestion
Once you have [added a dependency to your model](#write-dependencies-into-your-module), you can use SDK methods on the resource client.
For example:
```

Reply:
sounds good